portal-web/test/functional/com/liferay/portalweb/tests/enduser/collaboration/blogs/MyBlogTest.testcase

```diff
@@ -0,0 +1,68 @@
+@component-name = "portal-lima"
+definition {
+
+	property testray.main.component.name = "Blogs";
+
+	setUp {
+		TestCase.setUpPortalInstance();
+
+		User.firstLoginPG();
+
+		Navigator.openURL();
+
+		task ("Given the Blogs Portlet is added to the page") {
+			JSONLayout.addPublicLayout(
+				groupName = "Guest",
+				layoutName = "Blogs Page");
+
+			JSONLayout.addWidgetToPublicLayout(
+				column = 1,
+				groupName = "Guest",
+				layoutName = "Blogs Page",
+				widgetName = "Blogs");
+
+			Navigator.gotoPage(pageName = "Blogs Page");
+		}
+	}
+
+	@description = "Can Delete a Blog"
+	@priority = 4
```

Comment:
Why set this test as priority 4? Once the scenario have been defined and it is identified as critical(FP5), it indicates that one of the main functions of the feature in question should't brake no way, the actions related to CRUD (Create - Read - Update - DELETE) must be of utmost importance.

If you have questions or want to discuss the topic further, you can send me a message on slack